README.md

```diff
@@ -204,16 +206,21 @@ See below for information on how to [override ArcGIS styles](#overriding-arcgis-
 
 Of course, you don't need to use esri-loader to load the styles. See the [ArcGIS API for JavaScript documentation](https://developers.arcgis.com/javascript/latest/guide/get-api/index.html) for more information on how to load the ArcGIS styles by more traditional means such as adding `<link>` tags to your HTML, or `@import` statements to your CSS.
 
-## Why is this needed?
+## Do I need esri-loader?
+
+As of version v4.18 of the ArcGIS API for JavaScript you can try installing [@arcgis/core](https://www.npmjs.com/package/@arcgis/core) and [building with ES Modules](https://developers.arcgis.com/javascript/latest/guide/es-modules) and _instead_ of using esri-loader. It's also pretty easy to [migrate applications built  with esri-loader](https://developers.arcgis.com/javascript/latest/guide/es-modules/#esri-loader).
 
-Unfortunately, you can't simply `npm install` the ArcGIS API and then `import` 'esri' modules in a non-Dojo application. The only reliable way to load ArcGIS API for JavaScript modules is using Dojo's AMD loader. However, when using the ArcGIS API in an application built with another framework, you typically want to use the tools and conventions of that framework rather than the Dojo build system.
+<p id="why-is-this-needed">Unfortunately, prior to version 4.18, you couldn't simply `npm install` the ArcGIS API and `import` it's modules. The only reliable way to load ArcGIS API for JavaScript modules was using Dojo's AMD loader. Rather than using Dojo to build your applicaiton, esri-loader provides a way to dynamically load modules at runtime from a hosted build of the ArcGIS API (such as the <a href="https://developers.arcgis.com/javascript/latest/guide/get-api/#cdn">CDN</a>) into applications built using modern tools and framework conventions.</p>
```

Comment:
I think it's still worth mentioning here that some users might want to use esri-loader to take advantage of the fast cached CDN for their apps. That could still be a consideration.

Reply:
Minor typo: "build your applicaiton" > application